mne/evoked.py

```diff
@@ -901,24 +912,8 @@ def combine_evoked(all_evoked, weights):
     bads = list(set(evoked.info['bads']).union(*(ev.info['bads']
                                                  for ev in all_evoked[1:])))
     evoked.info['bads'] = bads
-
     evoked.data = sum(w * e.data for w, e in zip(weights, all_evoked))
-    # We should set nave based on how variances change when summing Gaussian
-    # random variables. From:
-    #
-    #    https://en.wikipedia.org/wiki/Weighted_arithmetic_mean
-    #
-    # We know that the variance of a weighted sample mean is:
-    #
-    #    σ^2 = w_1^2 σ_1^2 + w_2^2 σ_2^2 + ... + w_n^2 σ_n^2
-    #
-    # We estimate the variance of each evoked instance as 1 / nave to get:
-    #
-    #    σ^2 = w_1^2 / nave_1 + w_2^2 / nave_2 + ... + w_n^2 / nave_n
-    #
-    # And our resulting nave is the reciprocal of this:
```

Comment:
can you keep this block of text to explain the weight computation?

Reply:
Sure I'll add that back in tomorrow morning first thing.